libethcore/Precompiled.cpp

```diff
@@ -132,15 +132,34 @@ ETH_REGISTER_PRECOMPILED(modexp)(bytesConstRef _in)
 	return {true, ret};
 }
 
+namespace
+{
+	bigint expLengthAdjust(bigint _baseLength, bigint _expLength, bytesConstRef _in)
+	{
+		size_t const numConsideredBytes(min(_expLength, {32}));
```

Comment:
OK.  That's closer to what I wrote first.

I also think I should change `parseBigEndianRightPadded` to take `bigint` instead of `size_t`.

I'll make these changes after adding tests.